zerver/openapi/zulip.yaml

```diff
@@ -4281,6 +4281,21 @@ paths:
                                             **Changes**: New in Zulip 9.0 (feature level 266). Previously
                                             `realm_create_private_stream_policy` field used to control the
                                             permission to create private channels.
+                                    can_create_web_public_channel_group:
+                                      allOf:
+                                        - $ref: "#/components/schemas/GroupSettingValue"
+                                        - description: |
+                                            A [group-setting value](/api/group-setting-values) defining
+                                            the set of users who have permission to create web-public
+                                            channels in this organization.
+
+                                            This setting can only be set to `"role:moderators"`,
+                                            `"role:administrators"`, `"role:owners"` and
+                                            `"role:nobody"` system groups.
```

Comment:
I think this text is not necessary; we should instead be saying that the `server_supported_permission_settings` specifies what the valid values are for settings like this one.

The reason is that changes in the set of permitted values should not be thought of as a change in the API -- just a change in data. A correctly written client should be parsing `server_supported_permission_settings` and not require any changes if we were to adjust the set of valid values here.

Reply:
We can clean this up in a follow-up, since we don't have a documentation section in the group-setting-values page regarding  `server_supported_permission_settings` to link to yet.